app/components/private-cloud/sections/QuotasBudgetEstimation.tsx

```diff
@@ -1,4 +1,5 @@
-import { IconArrowNarrowRight, IconCurrencyDollarCanadian } from '@tabler/icons-react';
+import { Alert } from '@mantine/core';
+import { IconArrowNarrowRight, IconCurrencyDollarCanadian, IconExclamationCircle } from '@tabler/icons-react';
```

Comment:
[SonarCloud is reporting the IconCurrencyDollarCanadian import is not used](https://sonarcloud.io/project/issues?id=bcgov_platform-services-registry&pullRequest=6570&issueStatuses=OPEN,CONFIRMED&sinceLeakPeriod=true) and should be removed.